libs/platform/user/ebpf_platform_user.cpp

```diff
@@ -481,6 +487,25 @@ ebpf_queue_non_preemptible_work_item(_In_ ebpf_non_preemptible_work_item_t* work
     return false;
 }
 
+void
+ebpf_queue_preemptible_work_item(_In_ ebpf_preemptible_work_item_t* work_item)
+{
+    UNREFERENCED_PARAMETER(work_item);
+    // TODO: Add support for creating threads to queue preemptible work items in user mode.
```

Comment:
Why not use [NT thread pool API](https://docs.microsoft.com/en-us/windows/win32/api/threadpoolapiset/nf-threadpoolapiset-createthreadpoolwork)'s?

Reply:
Created issue #854